Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11.4-slim
+LABEL mantainer="volodymyr_dolgyi@ukr.net"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt

```

Comment:
The `COPY requirements.txt requirements.txt` command is correct, but you might want to ensure that `requirements.txt` is in the same directory as the Dockerfile. If it's in a different directory, you need to adjust the path accordingly.